kie-spring-boot/kie-spring-boot-autoconfiguration/kie-server-spring-boot-autoconfiguration-audit-replication/src/main/java/org/kie/server/spring/boot/autoconfiguration/audit/replication/AuditDataReplicationProcessEventProducer.java

```diff
@@ -21,24 +21,11 @@
 import org.jbpm.process.audit.AbstractAuditLoggerAdapter;
 import org.jbpm.process.audit.NodeInstanceLog;
 import org.jbpm.process.audit.ProcessInstanceLog;
-import org.kie.api.event.process.ProcessAsyncNodeScheduledEvent;
-import org.kie.api.event.process.ProcessCompletedEvent;
-import org.kie.api.event.process.ProcessNodeLeftEvent;
-import org.kie.api.event.process.ProcessNodeTriggeredEvent;
-import org.kie.api.event.process.ProcessStartedEvent;
-import org.kie.api.event.process.ProcessVariableChangedEvent;
-import org.kie.api.event.process.SLAViolatedEvent;
+import org.kie.api.event.process.*;
```

Comment:
Imho, adding an asterisk isn't a good practice, as the reader loses the target classes used here, but it's just an opinion.